client/my-sites/domains/domain-management/edit/navigation/style.scss

```diff
@@ -15,7 +15,7 @@
 			color: var( --color-neutral-90 );
 			small {
 				color: var( --color-neutral-50 );
-				font-size: 100%;
+				font-size: 14px;
```

Comment:
I think the "proper" way would be to use the predefined/named variables we have for the font sizes. In this case, it'd be `font-body-small`. See:
https://github.com/Automattic/wp-calypso/blob/efce6b162c3d6520c3b2a2dfa6e414161def75c9/client/assets/stylesheets/shared/_typography.scss#L49

And then this line would simply be `font-size: $font-body-small;`